File: app/src/main/java/com/loopeer/codereader/utils/ThemeUtils.java
package com.loopeer.codereader.utils;

import android.content.Context;
import android.support.v7.app.AppCompatDelegate;

public class ThemeUtils {
    public static final String THEME_DAY = "Default";
    public static final String THEME_NIGHT = "Night";

    @AppCompatDelegate.NightMode
    public static int getCurrentNightMode(Context context) {
        return PrefUtils.getPrefTheme(context).equals(THEME_DAY)
                ? AppCompatDelegate.MODE_NIGHT_NO
                : AppCompatDelegate.MODE_NIGHT_YES;
    }
}

File: code-reader-kt/src/main/java/com/loopeer/codereaderkt/utils/ThemeUtils.kt
package com.loopeer.codereaderkt.utils

import android.content.Context
import android.support.v7.app.AppCompatDelegate


object ThemeUtils {
    val THEME_DAY = "Default"
    val THEME_NIGHT = "Night"

    @AppCompatDelegate.NightMode
    fun getCurrentNightMode(context: Context): Int {
        return if (PrefUtils.getPrefTheme(context) == THEME_DAY)
            AppCompatDelegate.MODE_NIGHT_NO
        else
            AppCompatDelegate.MODE_NIGHT_YES
    }
}
